.github/workflows/validate.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: Validate
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check-version:
+    name: "Check Enos version"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: "Verify Enos version is released & latest"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          ver1=$(grep "const latestVersion" enos.js|awk '{print $4}'| tr -d ';')
+          ver2=$(grep "default" action.yml| awk '{print $2}')
```

Comment:
I like the idea of this but if feels super brittle. I would rather we cover it with two javascript tests that cover:
* Is the value of `latestVersion` an actual artifact that we can download. (This might already exist, if not we can reuse the asset download function to whip it up rather quick)
* Decode the action.yml into an Object/Map and ensure that the value of set for version matches latestVersion.

Then, it would be nice if pull requests triggered a workflow that installs node, builds and tests it.

Reply:
yep, sorry, this was meant to be a draft, playing around with ideas